x-pack/solutions/observability/plugins/apm/common/agent_configuration/setting_definitions/edot_sdk_settings.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import type { RawSettingDefinition } from './types';
+
+export const edotSDKSettings: RawSettingDefinition[] = [
+  {
+    key: 'my_test_config',
```

Comment:
@jackshirazi Please update the config options available with EDOT Java here

Reply:
instrumentation (added here) and recording added in general_settings file). I thought I had done context_propagation_only but I only did traces, not metrics and log sending, so we should leave that for now - unless we're doing this change in advance, in expectation the full set will be done?